### ISSUE TITLE
No more auto-print of results if "No anomalies detected."

### ISSUE THREAD

**Issue by BrashEQLibrium** [twitter/AnomalyDetection#35]
Suppose I have a twelve-month-periodic vector `vec`, for which there are no anomalies. The following should not happen automatically, but currently it does.

`anomaly_detection_obj <- AnomalyDetectionVec(vec, period = 12, threshold = "p95")`
`# [1] "No amomalies detected."

At the very least, whether to auto-print this message should be an argument in the function.

**Comment by owenvallis** [comment]
Good point. We can remove the print, and add a verbose mode for debugging.

**Comment by erikriverson** [comment]
@owenvallis Thank you! Also, you can use ?message to display messages from your functions when verbose is set, instead of ?print. Note that print actually returns a value, unlike message which is used for its side-effect of displaying a message to the user.

**Comment by jhochenbaum** [comment]
@erikriverson Ah thats good to know! We can definitely do that...

**Comment by BrashEQLibrium** [comment]
+1 to using message rather thant print

**Comment by BrashEQLibrium** [comment]
BTW, any ETA on when this change will be made? Seems like a straightforward replacement of a call to print with a call to message

**Comment by akejariwal** [comment]
@BrashEQLibrium Please submit a patch. We shall review and merge.

cc @owenvallis @jhochenbaum

**Comment by owenvallis** [comment]
@BrashEQLibrium This was added a while back. The current version has all the prints replaced with the new message call.
